.travis.yml

```
@@ -26,3 +30,8 @@ before_script:
 install:
   - 'rm ${BUNDLE_GEMFILE}.lock'
   - 'eval bundle install --jobs=3 --retry=3 --path=${BUNDLE_PATH:-$PWD/vendor/bundle}'
+
+jobs:
+  include:
+    - stage: build
+      script: docker build -t monitor_ci .
```

```suggestion
      script: docker build -t gredit/app .
```
Aunque no sirva para nada porque no se pushea =)